### ISSUE TITLE
what's the training data list ?

### ISSUE THREAD

**Issue by cliffgao** [kevinmicha/ANTIPASTI#11]
In the paper,  “We split 634  available correlation maps with corresponding log_10(K_D) values into a training set and a test set in a 95-5 ratio, repeating the split 5 times.”  
 However, I find that there are 701 PDB files in /data/dccm_maps_full_ags_all.  Could you please upload the  634 protein names list, which was used to train the model?  Thank you so much!
